app/Chat.tsx

```
@@ -450,48 +412,186 @@ What's on your mind? Let's dive in. 🌱`,
+
+          case 'honcho':
+            // Update the thought with honcho response
+            const updatedThought =
+              thoughtText + '\n\nHoncho Dialectic Response:\n\n' + chunk.text;
+            setThought(updatedThought);
+            break;
+
+          case 'pdf':
+            // Update the thought with PDF response
+            const updatedThoughtWithPDF =
+              thoughtText + '\n\nPDF Analysis:\n\n' + chunk.text;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)